src/containers/Generator/Generator.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux';
+import { setActivePalette } from '../../actions';
+import PropTypes from 'prop-types';
+import '../../main.scss';
+
+class Generator extends Component {
+  constructor() {
+    super()
+    this.state = {
+      locked: [],
+    }
+  }
+
+  componentDidMount() {
+    this.generatePalette()
+  }
+  
+  createColor = () => {
+    const options = '0123456789ABCDEF'
+    let hexArray = ['#']
+    for (let i = 0; i < 6; i++) {
+      let random = Math.floor(Math.random() * (16))
+      hexArray.push(options[random])
+    }
+    return hexArray.join('')
```

Comment:
Nice job on this, using a pre-made string is a little cumbersome but it is clean and easy to read 👍